### PR TITLE
test: Fix test helper

### PR DESCRIPTION
**Pull request by avniculae**
---

<a href="https://app.devin.ai/review/aave/aave-v4/pull/1200" target="_blank">
  <picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://static.devin.ai/assets/gh-open-in-devin-review-dark.svg?v=1">
    Open with Devin
  </picture>
</a>

### CODE REVIEW

File: tests/Base.t.sol [hunk 1]
```diff
@@ -2066,7 +2066,7 @@ abstract contract Base is Test {
     );
     uint256 targetTotalDebtValue = totalAdjustedCollateralValue.wadDivUp(desiredHf);
     require(
-      userAccountData.totalDebtValueRay / WadRayMath.RAY < targetTotalDebtValue,
+      userAccountData.totalDebtValueRay / WadRayMath.RAY <= targetTotalDebtValue,
```
Comment (yan-man) [inline tests/Base.t.sol:2069]: would it be better to make it `userAccountData.totalDebtValueRay` vs `targetTotalDebtValue * WadRayMath.RAY` to eliminate the precision loss due to division

Comment (avniculae) [reply tests/Base.t.sol:2069]: yes, I just wanted to keep alignment with logic below as well. I think if fuzzer finds such edge cases, then more things will break

**Comment by avniculae** [comment]
implemented this in #1218
